configs/prod/roles/patch.json

```diff
@@ -4,12 +4,35 @@
       "name": "Patch administrator",
       "description": "Perform any available operation against any Patch resource.",
       "system": true,
-      "platform_default": true,
-      "version": 5,
+      "platform_default": false,
+      "admin_default": true,
+      "version": 6,
       "access": [
         {
           "permission": "patch:*:*"
         },
+        {
+          "permission": "inventory:*:read"
+        },
+        {
+          "permission": "remediations:*:read"
+        },
+        {
+          "permission": "remediations:*:write"
+        }
+      ]
+    },
+    {
+      "name": "Patch viewer",
+      "description": "Read any Patch resource.",
+      "system": true,
+      "platform_default": true,
```

Comment:
@psegedy just for confirmation - `platform_default: true` means that role is given to everyone by default,

is this correct ?